frontend/src/registry/App.vue

```diff
@@ -17,10 +17,195 @@ export default {
 </script>
 
 <style>
-/* @import url('https://unpkg.com/bootstrap@3.3.7/dist/css/bootstrap.min.css');
 @import url('https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css');
 @import '../../../gwells/static/gwells/css/ie10-viewport-bug-workaround.css';
 @import '../../../gwells/static/gwells/css/main.css';
-@import '../../../gwells/static/gwells/css/custom-form.css';
-@import '../../../gwells/static/gwells/css/custom-main.css'; */
+@import '../../../gwells/static/gwells/css/custom-main.css';
+
+.row.no-pad {
+  margin-right:0;
+  margin-left:0;
+}
+.row.no-pad > [class*='col-'] {
+  padding-right:0;
+  padding-left:0;
+}
+.panel.no-pad {
+  margin-right:0;
+  margin-left:0;
+  padding-right:0;
+  padding-left:0;
+  margin-top:0;
+  padding-top:0;
+}
+.panel.no-pad > .panel-body {
+  margin-right:0;
+  margin-left:0;
+  padding-right:0;
+  padding-left:0;
+  margin-top:0;
+  padding-top:0;
+}
+.container-fluid.no-pad {
+  margin-right:0;
+  margin-left:0;
+  padding-right:0;
+  padding-left:0;
+}
+.registry-panel-title {
+  margin-left: 10px;
+  margin-top: 0px;
+  padding-top:0px;
+}
+.form-spacing {
+  margin-bottom: 15px;
+}
+.registry-disabled-item {
+  color: #808080;
+  cursor: auto!important;
+}
+.register-legal {
+  margin-top: 25px;
+}
+.btn {
+    display: inline-block;
+    padding: 7px 12px 5px 12px;
+    margin-bottom: 0;
+    font-size: 14px;
+    font-weight: normal;
+    line-height: 1.42857143;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: middle;
+    -ms-touch-action: manipulation;
+    touch-action: manipulation;
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    background-image: none;
+    border: 1px solid;
+    border-radius: 0px;
+}
+.btn-primary {
+    color: #fff;
+    background-color: #38598a;
```

Comment:
Can we look at using something like Sass? For example for defining variables, e.g. then we can change colours in a single location.